### DISCUSSION TITLE
blog/the_second_high_school/second_grade/up_term/11_week

### DISCUSSION THREAD

**Discussion by giscus** [yyyytawa/yyyytawa.github.io#12] [Announcements]
# blog/the_second_high_school/second_grade/up_term/11_week

日期: 开始: 20241117 星期日 结束: 202411xx 星期六 本周内容:

https://docs.yyyyt.top/blog/the_second_high_school/second_grade/up_term/11_week.html

**Comment by runoneall** [comment]
本周内容：

“’NoneType’ object is not callable” （（（
